pkg/skaffold/docker/parse.go

```diff
@@ -220,8 +220,13 @@ func processCopy(workspace string, value *parser.Node, paths map[string]struct{}
 		if hasMultiStageFlag(value.Flags) {
 			return nil
 		}
-		dep := path.Join(workspace, src)
-		paths[dep] = struct{}{}
+		if !strings.HasPrefix(src, "http://") && !strings.HasPrefix(src, "https://") {
```

Comment:
Maybe use url.Parse and check the Scheme? instead of checking both of these directly?

Reply:
this is one less step, since i'd have to do this string comparison on the scheme anyways right?

Reply:
Yeah I meant more of checking whether or not there is a scheme, but that function doesn't really make sense to me now that I look at it. This is fine.